app/src/screen.c

```diff
@@ -454,6 +466,90 @@ update_texture(struct screen *screen, const AVFrame *frame) {
     }
 }
 
+bool
+screen_render_text(struct screen *screen) {
```

Comment:
This opens the font and draws the glyphs on every frame :/

IMO, it should create the texture and render the text the first time, then cache it (to be invalidated on resize though).

Reply:
Agree.